ocs_ci/helpers/dr_helpers.py

```diff
@@ -73,3 +73,117 @@ def relocate(preferred_cluster, drpc_name, namespace):
     drpc_obj.wait_for_phase(constants.STATUS_RELOCATED)
 
     config.switch_ctx(prev_index)
+
+
+def get_drpc_name(namespace):
+    """
+    Get the DRPC Resource Name
+
+    Args:
+        namespace (str): Name of namespace
+
+    Returns:
+        str: DRPC resource name
+    """
+
+    drpc_obj = ocp.OCP(kind=constants.DRPC, namespace=namespace,).get()[
+        "items"
+    ][0]
+    return drpc_obj["metadata"]["name"]
+
+
+def get_drpolicy_name(namespace):
+    """
+    Get DRPolicy Name
+
+    Args:
+        namespace (str) : Name of namespace
+
+    Returns:
+        str: DRPolicy name
+    """
+
+    drpolicy_obj = ocp.OCP(kind=constants.DRPOLICY, namespace=namespace,).get()[
+        "items"
+    ][0]
+    return drpolicy_obj["metadata"]["name"]
+
+
+def get_primary_cluster_name(namespace):
+    """
+    Get Primary Cluster Name based on Namespace
+
+    Args:
+        namespace (str): Name of the Namespace
+
+    Returns:
+        str: Primary Cluster Name
+    """
+    config.switch_acm_ctx()
+    drpc_resource_name = get_drpc_name(namespace=namespace)
+    drpc_obj = ocp.OCP(
+        kind=constants.DRPC,
+        namespace="busybox-workloads-2",
+        resource_name=drpc_resource_name,
+    ).get()
+
+    if drpc_obj.get("spec").get("action") == constants.ACTION_FAILOVER:
+
+        cluster_name = drpc_obj["spec"]["failoverCluster"]
+
+    elif drpc_obj.get("spec").get("action") == constants.ACTION_RELOCATE:
+        cluster_name = drpc_obj["spec"]["preferredCluster"]
+
+    else:
+        cluster_name = drpc_obj["spec"]["preferredCluster"]
+
+    return cluster_name
+
+
+def set_primary_cluster_context(namespace):
```

Comment:
Wouldn't `MultiClusterConfig` class be a better place for these methods related to cluster context?

Reply:
@sidhant-agrawal , I feel a seperate helpers would be better. Because, MutlicClusterConfig is no more a specific class to handle RDR configs instead it is applicable to Managed services and may be many other multicluster scenarios in the future. So, ACM, Primary cluster, secondary clusters may be specific to RDR only. WDYT ?

Reply:
Make sense, these things will be specific to DR.